daemon/src/setup_contract.rs

```diff
@@ -452,7 +451,7 @@ pub async fn roll_over(
             rollover_params.quantity,
             rollover_params.leverage,
             n_payouts,
-            rollover_params.funding_fee,
+            rollover_params.funding_fee.clone().into(),
```

Comment:
Here we have to pass in the accumulated fees - see my comment on the representation of `RolloverFee`.

Reply:
We don't do this in this PR. I really do not wish to expand this PR scope even further.

Reply:
I'll pass it here after all now,  but we might need to sit together to tie the calculations down :)